data_tool/create.sh

```diff
@@ -0,0 +1,152 @@
+# Show the usage screen, since no arguments were given
```

Comment:
Shell scripts should always have a shebang (`#!/usr/bin/env bash` interpreter directive) at the top so that they run using the right shell. People trying to execute these scripts with their shell set to `zsh` or similar will encounter lots of errors (as I did).

The same applies to Python scripts intended to be used as executables (i.e. `#!/usr/bin/env python3`) so it knows to use Python (and which version).

In general, I think it's best to avoid writing Bash scripts for anything complex (i.e. anything more than a wrapper around a couple of commands). Python is installed by default on pretty much all distros, and it has things like `argparse` and a decent standard library for performing system-level tasks.